File: Homework 5/C++/CheckAlphaString.cpp
#include "CheckAlphaString.h"

CheckAlphaString::CheckAlphaString() {

}

CheckAlphaString::~CheckAlphaString() {

}

bool CheckAlphaString::check(string input) {

	for (int i = 0; i < input.length(); ++i) {

		if (!isalpha(input[i])) {

			return false;

		}

	}

	return true;

}

File: Homework 5/Java/CheckAlphaString.java
/*
 * Objective: check to see if alphabetic or numeric, return false if numeric 
*/

public class CheckAlphaString {

	public static boolean check(String input) {
		int i;
		char[] inputArray = input.toCharArray();
		for (i = 0; i < input.length(); i++) {
			if (!(Character.isLetter(inputArray[i]))) 
				return false;

		}
		return true;

	}
}
